test/furtherTests.js

```diff
@@ -57,22 +41,83 @@ contract("Further tests", function(accounts) {
   it("Total Supply", async function() {
     let supply = await master.totalSupply();
     assert(web3.utils.fromWei(supply) > 0, "Supply should not be 0"); //added miner
-    assert(web3.utils.fromWei(supply) < 200000, "Supply should be less than 100k"); //added miner
+    assert(
+      web3.utils.fromWei(supply) < 200000,
+      "Supply should be less than 100k"
+    ); //added miner
   });
+
   it("Test Changing Dispute Fee", async function() {
-    await  master.theLazyCoon(accounts[6], web3.utils.toWei("5000", "ether"))
-    await master.theLazyCoon(accounts[7], web3.utils.toWei("5000", "ether"))
+    await master.theLazyCoon(accounts[6], web3.utils.toWei("5000", "ether"));
+    await master.theLazyCoon(accounts[7], web3.utils.toWei("5000", "ether"));
     var disputeFee1 = await master.getUintVar(
       web3.utils.keccak256("disputeFee")
     );
-    newOracle = await Tellor.new();
-    await master.changeTellorContract(newOracle.address)
-    await master.depositStake({from:accounts[6]})
-    await  master.depositStake({from:accounts[7]})
+    // newOracle = await Tellor.new();
+    // await master.changeTellorContract(newOracle.address)
+    await master.depositStake({ from: accounts[6] });
+    await master.depositStake({ from: accounts[7] });
     assert(
       (await master.getUintVar(web3.utils.keccak256("disputeFee"))) <
         disputeFee1,
       "disputeFee should change"
     );
   });
+
```

Comment:
This test needs to be re rethought... It's not easy to programmatically verify that tips are being burned when we have time based rewards. Sure, we can bend the assertions to make it pass, but it would make the tests a bit useless